### ISSUE TITLE
libsysfs.pc missing

### ISSUE THREAD

**Issue by Earnestly** [dankamongmen/omphalos#2]
The addition of `libsysfs.pc` is debian only making it pretty difficult to workaround on other systems however adding the following does allow the configure to succeed:

```
./configure LIBSYSFS_CFLAGS=-I/usr/include/sysfs LIBSYSFS_LIBS=-lsysfs …
```

**Comment by dankamongmen** [comment]
Arch also lacks libsysfs. I really ought pull this out.

**Comment by dankamongmen** [comment]
Looks like this file is installed as a Debian original, heh.

**Comment by dankamongmen** [comment]
hey @Earnestly , thanks for these early bug reports on arch. i just started running arch recently on my laptop, and am planning on packaging this tool up for the AUR.

**Comment by dankamongmen** [comment]
Ideally we would have dropped all use of garbage libsysfs, but that's another bug for a more idealistic day. Destroy all culture! Resolved in 0.99.8.
